apple/OmnivoreKit/Sources/App/Views/Labels/LabelsView.swift

```diff
@@ -114,10 +114,7 @@ struct CreateLabelView: View {
       .padding(.bottom, 8)
 
       TextField("Label Name", text: $newLabelName)
-      #if os(iOS)
-        .keyboardType(.alphabet)
-      #endif
-      .textFieldStyle(StandardTextFieldStyle())
+        .textFieldStyle(StandardTextFieldStyle())
```

Comment:
Do we need to explicitly use `.default` keyboard type?

Reply:
@sywhb my understanding is we do not need to be explicit, unless you saw something in docs.